pycloudlib/oci/utils.py

```diff
@@ -84,14 +82,126 @@ def get_subnet_id_by_name(
     return subnets[0].id
 
 
+def _get_subnet_features(
+    subnet: oci.core.models.Subnet,
+) -> Dict[str, Any]:
+    """
+    Get the availability domain, private, and networking type of the subnet.
```

Comment:
This could be reworded for clarity; e.g. "get the private of the subnet" doesn't make a ton of immediate sense.